_posts/2016-09-10-stripe-dress.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: The psychology of perception tests an age-old fashion foible
+author: neuroamanda
+image:
+  feature: 2016-09-10-stripe-dress.jpg
+  credit: Gareth Williams
+  caption: Horizontal-striped dress
+---
+
+_Horizontal stripes on clothing are in fact slimming_
+
+[The dress](https://en.wikipedia.org/wiki/The_dress) sparked the hottest fashion debate of the 21st century, but another mystery has occupied scientists and fashionistas alike since the mid-19th century: do horizontal stripes make you look fat?
```

Comment:
Link to https://figureoneblog.wordpress.com/2015/02/27/what-the-media-missed-about-thedress/ instead?

Also, rather than saying "the hottest fashion debate", maybe say "one of the hottest fashion debates" -- unless you know of a scale for measuring said hotness when it comes to debates. :P